pkg/security/probe/discarders.go

```diff
@@ -84,7 +86,7 @@ var (
 )
 
 var (
-	dentryInvalidDiscarder = []interface{}{dentryPathKeyNotFound}
+	dentryInvalidDiscarder = []interface{}{""}
```

Comment:
I think we can now remove this whole "invalid" discarder mechanism, see https://github.com/DataDog/datadog-agent/blob/main/pkg/security/probe/discarders.go#L355

This can be done in another PR that is being said